### PR TITLE
Prefix headers are moved to common file

### PR DESCRIPTION
**Pull request by olimpias**
In #254 PR, decided to move prefix http to common file. They are moved to network.

### CODE REVIEW

**Comment by ldemailly** [comment]
fix the compile failures?
usually it is best to build and test before PR

/ok-to-test

**Comment by olimpias** [comment]
Yes normally I do that but I don't know why this time I jumped to build state. I thought there wont be a problem. I shouldn't have rushed. that is a lesson to me :).

File: fnet/network.go [hunk 1]
```diff
@@ -25,6 +25,19 @@ import (
 	"istio.io/fortio/version"
 )
 
+const (
+	// DefaultGRPCPort is the Fortio gRPC server default port number.
+	DefaultGRPCPort = "8079"
+	// DefaultHTTPPort is the Fortio http server default port number.
```
Comment (ldemailly) [inline fnet/network.go]: that comment isnt correct 
call 80 and 443 StandardHTTP(S)Port and the comment is it is the standard

unlike 8079 which is the default port the grpc server listens to (the default http is 8080)

**Review by ldemailly — COMMENTED** [review]
can you change the variable name too, grpc is “default” but http/https are standard ports (they are not the same meaning so they shouldn’t be called the same)

**Review by ldemailly — APPROVED** [review]
thanks

**Pull request merged**